pages/using_elasticsearch.md

```diff
@@ -45,3 +52,30 @@ In production, JHipster expects an external Elasticsearch instance. By default,
 ### Using on Heroku
 
 On Heroku, the [Bonsai Elasticsearch](https://elements.heroku.com/addons/bonsai) is configured as an add-on. JHipster is automatically configured to talk to it. 
+
+Unfortunately, as of JHipster 7.9.3, Elasticsearch [doesn't work out of the box with Heroku](https://github.com/jhipster/generator-jhipster/issues/20315). To solve this, you can create a Docker image with Elasticsearch and deploy it to somewhere that can run it, or you can use Elastic Cloud. 
```

Comment:
Maybe one sentence why we don't describe the elastic heroku add-on? (because of pricing, no free trial?)